packages/backend/server/src/core/config.ts

```diff
@@ -22,6 +22,21 @@ export function ADD_ENABLED_FEATURES(feature: ServerFeature) {
   ENABLED_FEATURES.add(feature);
 }
 
+@ObjectType()
+export class PasswordLimitsType {
+  @Field()
+  minLength!: number;
+
+  @Field()
+  maxLength!: number;
+}
+
+@ObjectType()
+export class AuthPolicyType {
```

Comment:
`CredentialsRequirementType`, `Policy` term might be used in further Permission module